src/components/include/utils/threads/message_loop_thread.h

```diff
@@ -147,9 +147,8 @@ MessageLoopThread<Q>::MessageLoopThread(const std::string& name,
 
 template <class Q>
 MessageLoopThread<Q>::~MessageLoopThread() {
-  Shutdown();
```

Comment:
@SNiukalov Why did you remove it?

Reply:
Shutdown () was removed because it is redundant. Destructor Thread will make this.